Add Mappers' Report: May/June news post (#3763)
ppy/osu-wiki: news/2020-06-26-mappers-report-May-June.md

---
layout: post
title: "The Mappers' Report: May/June"
date: 2020-06-30 12:00:00 +0000
---

> Have you missed us? I bet you did! Sorry for the wait but here we are again, the fourth and fifth issue of the Mappers' Report in one post! Full with news and happenings all over the mapping community in osu! Did your favorite mapper won a contest? Or got the long awaited ranking criteria change finally happen? — Jump right in and find out!

Reviewer: ```suggestion
Have you missed us? I bet you did! Sorry for the wait but here we are again, the fourth and fifth issue of the Mappers' Report in one post! Full with news and happenings all over the mapping community in osu! Did your favorite mapper win a contest? Or did the long awaited ranking criteria change finally happen? — Jump right in and find out!
```

![](/wiki/shared/news/banners/the-mappers-report.jpg)

This month's issue of the Mappers' Report contains following topics:

- **[Ranking Criteria Changes](#rc-changes)**
- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
- **[Mappers' Showcase](#mappers-showcase)**
- **[Mapping Contests](#mapping-contests)**
- **[Around the Community](#around-community)**
- **[Conclusion](#conclusion)**

## <a id="rc-changes"></a>Ranking Criteria Changes

As every month we keep you up to date with what happened in the Ranking Criteria. All rule and guideline changes are listed below and in addition you can see open proposals to give your feedback to them!

### All Modes

> - **[Remove glossaries from RC pages](https://osu.ppy.sh/community/forums/topics/1064466)**: The content of the glossaries will be put into own wiki pages! This happened already for the general Ranking Criteria and the osu! Ranking Criteria. The other will follow soon!

Reviewer: ```suggestion
- **[Remove glossaries from RC pages](https://osu.ppy.sh/community/forums/topics/1064466)**: The content of the glossaries will be put into individual wiki pages! This happened already for the general Ranking Criteria and the osu! Ranking Criteria. The others will follow soon!
```

Reviewer 2: is "the others will follow soon" even true whos gonna do that

Reviewer 3: > is "the others will follow soon" even true whos gonna do that

<s>Are you making a joke</s>

Reviewer 2: not a joke lol pishi did the general + osu! ones and I don't think he cares abt other modes' RC
I'm not gonna do it (soon) cuz I have already overcommitted time to wiki stuff

Author: I definitely will check that out soon for osu!mania and see what can be done.

> - **[Allow .ogg files as a main audio source](https://osu.ppy.sh/community/forums/topics/1021547)**: Always struggle with your map getting uploaded because of too large file size? Then we have good news! You may now use .ogg as audio to save some file size! Note that this still must follow the regular kbit/s rules!

Reviewer: ```suggestion
- **[Allow .ogg files as a main audio source](https://osu.ppy.sh/community/forums/topics/1021547)**: Always struggle with your map getting uploaded because of too large file size? Then we have good news! You may now use .ogg for the song audio to save some file size! Note that this still must follow the regular kbit/s rules!
```

### osu!

- **[Intuitive Reverse Arrow Movement](https://osu.ppy.sh/community/forums/topics/1033709?start=unread):** Reverse Arrows must now point into the direction in which the slider ball will move after hitting it.
- **[Restrictions on "accelerating" sliders](https://osu.ppy.sh/community/forums/topics/1060629?n=1):** A small clarification that the slider path must always be visible.

> - **[Circle size](https://osu.ppy.sh/community/forums/topics/1082308):** You asked, here it is! Long have it been that there was a ranke song with a higher circle cize than 7, now we will hopefully see some more coming soon! 

Reviewer 4: ```suggestion
- **[Circle size](https://osu.ppy.sh/community/forums/topics/1082308):** You asked, here it is! Long have it been that there was a ranked song with a higher circle cize than 7, now we will hopefully see some more coming soon! 
```

Reviewer: ```suggestion
- **[Circle size](https://osu.ppy.sh/community/forums/topics/1082308):** You asked, here it is! It's been a long time since there was a ranked map with a higher circle size than 7, now we will hopefully see some more coming soon! 
```

The Beatmap Nominators never rest, the Beatmap Nominators never sleep. Just kidding, of course they do, they are still humans. However, the group size is always changing! New are coming, old are going. This month was no exception!

### BN Additions

- **osu!:** [Teky](https://osu.ppy.sh/users/10520912), [DeviousPanda](https://osu.ppy.sh/users/4966334), [Stixy](https://osu.ppy.sh/users/9000308), [Trynna](https://osu.ppy.sh/users/2652951), [Fall](https://osu.ppy.sh/users/4800816), [Elayue](https://osu.ppy.sh/users/6400861) & [NexusQI](https://osu.ppy.sh/users/13822800)
- **osu!taiko:** [realy0\_](https://osu.ppy.sh/users/8863359), [\_DUSK\_](https://osu.ppy.sh/users/6092181), [davidminh0111](https://osu.ppy.sh/users/9623142), [Cychloryn](https://osu.ppy.sh/users/6921736) & [Axer](https://osu.ppy.sh/users/7299864)
- **osu!catch:** [Baron](https://osu.ppy.sh/users/10286499), [Secre](https://osu.ppy.sh/users/2306637) & [Bastian](https://osu.ppy.sh/users/6345176)
- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107), [Shima Rin](https://osu.ppy.sh/users/6089608), [Garalulu](https://osu.ppy.sh/users/757783) & [Muse Dash](https://osu.ppy.sh/users/13695676)

### BN Resignations

> - **osu!:** [Affirmation](https://osu.ppy.sh/users/6186628) & [wafer](https://osu.ppy.sh/users/9416836)

Reviewer: ```suggestion
- **osu!:** [Affirmation](https://osu.ppy.sh/users/6186628)
```

- **osu!mania:** [Drum-Hitnormal](https://osu.ppy.sh/users/748722), [\_Kobii](https://osu.ppy.sh/users/6209713) & [Unpredictable](https://osu.ppy.sh/users/7560872)

### BN and QAT/NAT Badges

Recently a new wave of badges acknowledging the length of time members have put into teams for managing ranked! Those who are on this list have been contributing to the community in so many ways during their time on osu!, and fully deserve this acknowledgement. We thank them all for their contribution so far and hope to see much more of them in the future! Here is the list of those:

- **BN 1 Year Badge:** [incandescence](osu.ppy.sh/users/6256027), [Kaitjuh](osu.ppy.sh/users/2225327), [Unpredictable](osu.ppy.sh/users/7560872), [Nuvolina](osu.ppy.sh/users/10974170), [Dored](osu.ppy.sh/users/10284894)
- **BN 2 Year Badge:** [JBHyperion](osu.ppy.sh/users/4879508), [Nevo](osu.ppy.sh/users/7451883), [Mordred](osu.ppy.sh/users/7265097), [bossandy](osu.ppy.sh/users/360437), [Namki](osu.ppy.sh/users/5248582)
- **BN 3 Year Badge:** [Benita](osu.ppy.sh/users/4023183), [Sonnyc](osu.ppy.sh/users/11771)
- **BN 4 Year Badge:**  [Nozhomi](osu.ppy.sh/users/2716981)

> - **NAT 1 Year Badges:** [Ascendance](osu.ppy.sh/users/2931883), [Noffy](osu.ppy.sh/users/1541323), [Faputa](osu.ppy.sh/users/845733)
> - **NAT 2 Year Badge:** [Lasse](osu.ppy.sh/users/896613), [-Mo-](osu.ppy.sh/users/2202163), [Chaoslitz](osu.ppy.sh/users/3621552)
> - **NAT 3 Year Badge:** [OnosakiHito](osu.ppy.sh/users/290128)

Reviewer: ```suggestion
- **NAT 1 Year Badges:** [Ascendance](osu.ppy.sh/users/2931883), [Noffy](osu.ppy.sh/users/1541323), [Faputa](osu.ppy.sh/users/845733)
- **QAT/NAT 2 Year Badge:** [Lasse](osu.ppy.sh/users/896613), [-Mo-](osu.ppy.sh/users/2202163), [Chaoslitz](osu.ppy.sh/users/3621552)
- **QAT 3 Year Badge:** [OnosakiHito](osu.ppy.sh/users/290128)
```

these were specified in the list I gave before to match up with their badge descriptions, for instance ono got the badge late but was never NAT

## <a id="mappers-showcase"></a>Mappers' Showcase

The Mappers' Showcase parades lesser-known mappers from every game mode. Since we bundled two issues in one post, this month will feature two game modes. 

> Thanks to our selector for osu!catch [Jemzuu](https://osu.ppy.sh/users/7890134) and our selectors for osu!mania [Scotty](https://osu.ppy.sh/users/11085809) & [Unpredictable](https://osu.ppy.sh/users/7560872)!

Reviewer: Could mention "Phob and dika312" for catch and  "BringoBrango and Ticle" for mania so it's clear who's what mode being showcased

Author: Good idea, added game mode behind the names to make it more clear

## <a id="mapping-contests"></a>Mapping Contests

Get your editor ready. The mapping contests are here! Whether you are a veteran or making your first steps in mapping, contests are for everyone! In this section you can see what contests have been held in May or are ongoing!

### Monthly Beatmapping Contests

The [Monthly Beatmapping Contests](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest) (MBC) gives to you your monthly dose of mapping competitions. Each month's contest features a different mapping limitation that entries must adhere to.

The April's contest results are out and we congratulate [hypercyte](https://osu.ppy.sh/users/9155377) for winning and getting his Tier 2 badge! Good job also to [fanzhen0019](https://osu.ppy.sh/users/418699) for being community vote runner-up and [Acyl](https://osu.ppy.sh/users/1943309) for being the screening crew favorite!

> The May and June Montly Beatmapping Contests have both their submission phase concluded and we can hope to see the results of it soon. Once they are out we make sure to let you know!

Reviewer: ```suggestion
The May and June Montly Beatmapping Contests have both of their submission phases concluded and we hope to see the results soon. Once they are out we'll be sure to let you know!
```

### osu! Mapping Olympiad #7: osu!mania

Keyboard mashers, rise up! It's your time to shine!
The [osu! Mapping Olympiad](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) is a returning mapping contest which features a different game mode with every version. And this time only for the osu!mania mappers out there!

Luring all mappers out there with Olympiad points for the top 3, which awards the Mapping Olympiad badge, and for the winner an unique user title, the "Elite Mapper"!

Do you have what it takes to be the next "Elite Mapper"? Then jump right into it, grab one of the pre-timed .osz and start to map! We're excited to see all the mania maps!

### 2020 osu!catch Mapping World Cup

> Fruit catcher, now's your chance! The 2020 edition of the [osu!catch Mapping World Cup](https://osu.ppy.sh/community/forums/topics/1089125) has arrived! If you are skilled with placing fruits in the editor get your 2 to 5 best friends by their hand and sign up! Can the Philippines keep their title as osu!catch Mapping World Champion or will an other country get the 1st place this time?

Reviewer: If this newspost is out after 6/27 the description should remove mention of signing up because signups will be closed

Author: Changed the description and removed the sign up mention as they will be closed already with the release of this Mappers' report issue

This contests has also something special. It can have two winners! One winner will be selected from the Taiwanese participants, the other winner will be selected by anyone. That's right, everyone is allowed to participate in the contest! 

So what are you waiting for, grab the pre-timed .osz and join in! Supporter and maybe even a profile badge is waiting for you!

Are there any other mapping contests going on? Or being announced soon? Please let us know! We would love to give it more attention to the public. No matter if it's mode or language specific. Head over to the [forum post](https://osu.ppy.sh/community/forums/topics/1027590) where you can find the submission form for it. 

We hope to see a lot of exciting mapping contests in the future!

## <a id="around-community"></a>Around the Community

### New Review System in Beatmap Discussion

Modders, listen. We have great news for you! As some may already noticed, modding got an update! The review system which got announced a while ago got finally added to the Beatmap Discussion! What it is you ask?

> Some may remember the old modding system which worked through the forum. People posted mods in mappers beatmap forum threads. The difference to the current modding system is that modder could bundle everything in one, huge post if they liked. A lot of modders liked it and still would like to have a similar modding possibility.
> 
> And now they finally have! You can now bundle your modding in one, big post! Just like you could in the old beatmap threads. To access it, go into the beatmap discussion of any beatmap you like to mod and then go to the "Review" tab. That's all! Now you can mod like in the old days. Just with all the features which the current modding system offers you!
> 
> You think it could be better and something is still missing? Or have you found any bug in it? The devs would love to hear that! Come join the [osu!dev Discord server](https://discord.gg/ppy) and join the discussion about the review system in the #modding channel! 

Reviewer: ```suggestion
Some may remember the old modding system which worked through the forum. People posted mods in mappers' beatmap forum threads. The difference to the current modding system is that modder could bundle everything in one, huge post if they liked. A lot of modders liked it and still would like to have a similar format when modding.

Now they finally do! You can now bundle your mods in one, big post! To access it, go into the beatmap discussion of any beatmap you like to mod and then go to the "Review" tab. That's all! Now you can mod in a format that lets you control what order the mod is read in, which is very useful for reoccurring or bigger issues in maps. Be sure to check out all the features which the current modding system offers to you!

Do you think it could be better and something is still missing? Or did you find a bug in it? The developrs would love to hear your feedback! Come join the [osu!dev Discord server](https://discord.gg/ppy) and join the discussion about the review system in the #modding channel! 
```

### The osu!wiki is hiring

Okay, hiring is maybe the wrong word for it. But on a serious note, the osu!wiki is a huge knowledge database which features topics about everything about osu! Not just mapping and/or modding related such as the entire Ranking Criteria or various guides for mapping, but also all kinds of information, such as the osu! team, various gameplay guides, contests and tournaments and much more.

But sadly it's run by only a few people and a lot of wiki entries are outdated. But good news: Everyone can help! And the people behind the osu!wiki would be surely happy to see more helping hands to update everything and provide the latest information all around osu! 

Contributing is not even that hard and you can get all help you need to get started in the #osu-wiki channel on the [osu!dev Discord server](https://discord.gg/ppy). Do not hesitate to ask there, they won't bite you!

<a id="conclusion"></a>

---

> Phew, that was a bit of a read, right? We're happy that you made it until here! Once again sorry for the long wait. We will work on our schedule to make sure future issues of the Mappers' Report going out in time. You can expect us in the future with the end of every month. Should really not that much happen in osu! we will probably make again one bigger news post, but we try to keep these on a minimum! And please don't forgot, should you have anything interesting and mapping or modding related for us to mention, make sure to tell it to [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323)!

Reviewer: ```suggestion
Phew, that was a bit of a read, right? We're happy that you made it until here! Once again sorry for the long wait. We will work on our schedule to make sure future issues of the Mappers' Report going out in time. You can expect us in the future with the end of every month. Should really not that much happen in osu! we will probably make again one bigger news post, but we try to keep these on a minimum! And please don't forget, should you have anything interesting and mapping or modding related for us to mention, make sure to tell it to [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323)!
```

![](/wiki/shared/news/banners/the-mappers-report.jpg)

This month's issue of the Mappers' Report contains following topics:

- **[Ranking Criteria Changes](#rc-changes)**
- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
- **[Mappers' Showcase](#mappers-showcase)**
- **[Mapping Contests](#mapping-contests)**
- **[Around the Community](#around-community)**
- **[Conclusion](#conclusion)**

## <a id="rc-changes"></a>Ranking Criteria Changes

As every month we keep you up to date with what happened in the Ranking Criteria. All rule and guideline changes are listed below and in addition you can see open proposals to give your feedback to them!

### All Modes

- **[Remove glossaries from RC pages](https://osu.ppy.sh/community/forums/topics/1064466)**: The content of the glossaries will be put into individual wiki pages! This happened already for the general Ranking Criteria and the osu! Ranking Criteria. The others will follow soon!

> - **[Allow .ogg files as a main audio source](https://osu.ppy.sh/community/forums/topics/1021547)**: Always struggle with your map getting uploaded because of too large file size? Then we have good news! You may now use .ogg for the song audio to save some file size! Note that this still must follow the regular kbit/s rules!

Reviewer 5: ```suggestion
- **[Allow .ogg files as a main audio source](https://osu.ppy.sh/community/forums/topics/1021547)**: Always struggle with getting your map uploaded, because of too large file size? Then we have good news! You can now use .ogg audio files to save some space! Note that they still have to follow the regular kbit/s rules!
```

### osu!

> - **[Intuitive Reverse Arrow Movement](https://osu.ppy.sh/community/forums/topics/1033709?start=unread):** Reverse Arrows must now point into the direction in which the slider ball will move after hitting it.

Reviewer 5: ```suggestion
- **[Intuitive Reverse Arrow Movement](https://osu.ppy.sh/community/forums/topics/1033709?start=unread):** Reverse Arrows now have to point into the direction in which the slider ball moves after hitting it.
```

> - **[Restrictions on "accelerating" sliders](https://osu.ppy.sh/community/forums/topics/1060629?n=1):** A small clarification that the slider path must always be visible.

Reviewer 5: ```suggestion
- **[Restrictions on "accelerating" sliders](https://osu.ppy.sh/community/forums/topics/1060629?n=1):** A small clarification that the slider path has to be always visible.
```

> - **[Circle size](https://osu.ppy.sh/community/forums/topics/1082308):** You asked, here it is! It's been a long time since there was a ranked map with a higher circle size than 7, now we will hopefully see some more coming soon! 

Reviewer 5: ```suggestion
- **[Circle size](https://osu.ppy.sh/community/forums/topics/1082308):** You asked for it and here it is! It's been a long time, since there was a ranked map with a circle size higher than 7. Now we will hopefully see some more coming soon! 
```

### osu!catch

> - **[Dashes and higher-snapped dashes on Rain's](https://osu.ppy.sh/community/forums/topics/1033696):** The RC change defined the allowed snap for dashes on Rain difficulties. This clarifies the definition of a higher-snapped dashes which was a problem for almost half a year.

Reviewer 5: ```suggestion
- **[Dashes and higher-snapped dashes on Rain's](https://osu.ppy.sh/community/forums/topics/1033696):** This RC change defines the allowed snap for dashes on Rain difficulties. It also clarifies the definition of a higher-snapped dashes which was a problem for almost half a year.
```

### osu!mania

> - **[Chord usage at 1/4 pattern in normal difficulties](https://osu.ppy.sh/community/forums/topics/1079778):** This change will allow to use chords as well at the beginning of 1/4 pattern in Normal and 1/2 pattern in Easy difficulties. Was only to end a pattern before this change.

Reviewer 5: ```suggestion
- **[Chord usage at 1/4 pattern in normal difficulties](https://osu.ppy.sh/community/forums/topics/1079778):** This change allows the usage of chords at the beginning of 1/4 patterns in Normal and 1/2 patterns in Easy difficulties. Before you could only use chords to end a pattern.
```

## Open Ranking Criteria Proposals

> There are various open Ranking Criteria proposal in the [forum](https://osu.ppy.sh/community/forums/87). Many of them are listed in the last issue of the [Mappers' Report](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april#rc-changes), but we also got some new one which you can find below!

Reviewer 5: ```suggestion
There are various open Ranking Criteria proposals in the [forum](https://osu.ppy.sh/community/forums/87). Many of them are listed in the last issue of the [Mappers' Report](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april#rc-changes), but we also have some new ones which you can find below!
```

> For all Modes:

Reviewer 5: ```suggestion
For all modes:
```

- **[Discussion to fix Vetoes](https://osu.ppy.sh/community/forums/topics/1068618)**
- **[Increase average bitrate limitation from 192kbs to 320kbs](https://osu.ppy.sh/community/forums/topics/1072965)**
- **[Spoiler in maps](https://osu.ppy.sh/community/forums/topics/1083667)**

For osu!:

- **[Make a limit for longest combo in osu!std&CTB map](https://osu.ppy.sh/community/forums/topics/1071402)**

For osu!mania:

- **[change chord guidelines in 4k normal/hard diffs](https://osu.ppy.sh/community/forums/topics/1067554)**
- **[Redefine the "stream" terminology in mania RC](https://osu.ppy.sh/community/forums/topics/1079647)**

## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT

> The Beatmap Nominators never rest, the Beatmap Nominators never sleep. Just kidding, of course they do, they are still humans. However, the group size is always changing! New are coming, old are going. This month was no exception!

Reviewer 5: ```suggestion
The Beatmap Nominators never rest, the Beatmap Nominators never sleep. Just kidding, of course they do, they are humans too. However, the group line-up is always changing! New people are coming, old are going away. This month was no exception!
```

### BN Additions

- **osu!:** [Teky](https://osu.ppy.sh/users/10520912), [DeviousPanda](https://osu.ppy.sh/users/4966334), [Stixy](https://osu.ppy.sh/users/9000308), [Trynna](https://osu.ppy.sh/users/2652951), [Fall](https://osu.ppy.sh/users/4800816), [Elayue](https://osu.ppy.sh/users/6400861), [NexusQI](https://osu.ppy.sh/users/13822800) & [Garden](https://osu.ppy.sh/users/2849992)
- **osu!taiko:** [realy0\_](https://osu.ppy.sh/users/8863359), [\_DUSK\_](https://osu.ppy.sh/users/6092181), [davidminh0111](https://osu.ppy.sh/users/9623142), [Cychloryn](https://osu.ppy.sh/users/6921736) & [Axer](https://osu.ppy.sh/users/7299864)
- **osu!catch:** [Baron](https://osu.ppy.sh/users/10286499), [Secre](https://osu.ppy.sh/users/2306637) & [Bastian](https://osu.ppy.sh/users/6345176)
- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107), [Shima Rin](https://osu.ppy.sh/users/6089608), [Garalulu](https://osu.ppy.sh/users/757783) & [Muse Dash](https://osu.ppy.sh/users/13695676)

> Do you want to see your own name on that list too? And do you like modding and helping other people with their beatmaps? Then become a Beatmap Nominator! The applications are always open. Head over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today!

Reviewer 5: dunno why but first three sentences sound incorrect to me, although I have no idea how to reword them properly

Author: Hmm, can't really see what exactly sounds wrong on it?

Reviewer 6: They sound alright to me but maybe they'd be a little better-sounding this way?

```suggestion
Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of those questions is "yes", then you can also become a Beatmap Nominator! The applications are always open. Head over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today!
```

Author: Okay yeah, sounds indeed a bit better. Gonna use that!

### BN Resignations

- **osu!:** [Affirmation](https://osu.ppy.sh/users/6186628)
- **osu!taiko:** [Cynplytholowazy](https://osu.ppy.sh/users/3901754) & [Gamelan4](https://osu.ppy.sh/users/9856910)
- **osu!catch:** [-Luminate](https://osu.ppy.sh/users/4778689)
- **osu!mania:** [\_Kobii](https://osu.ppy.sh/users/6209713) & [Unpredictable](https://osu.ppy.sh/users/7560872)

### BN and QAT/NAT Badges

> Recently a new wave of badges acknowledging the length of time members have put into teams for managing ranked! Those who are on this list have been contributing to the community in so many ways during their time on osu!, and fully deserve this acknowledgement. We thank them all for their contribution so far and hope to see much more of them in the future! Here is the list of those:

Reviewer 5: ```suggestion
Recently a new wave of badges acknowledging the time which group members have put into beatmap management teams! Those who are on this list have been contributing to the community in so many ways during their time on osu!, and fully deserve this acknowledgement. We are thankful for all their contributions and hope to see much more in the future! Here is the list of the awarded users:
```

- **BN 1 Year Badge:** [incandescence](https://osu.ppy.sh/users/6256027), [Kaitjuh](https://osu.ppy.sh/users/2225327), [Unpredictable](https://osu.ppy.sh/users/7560872), [Nuvolina](https://osu.ppy.sh/users/10974170) & [Dored](https://osu.ppy.sh/users/10284894)
- **BN 2 Year Badge:** [JBHyperion](https://osu.ppy.sh/users/4879508), [Nevo](https://osu.ppy.sh/users/7451883), [Mordred](https://osu.ppy.sh/users/7265097), [bossandy](https://osu.ppy.sh/users/360437) & [Namki](https://osu.ppy.sh/users/5248582)
- **BN 3 Year Badge:** [Benita](https://osu.ppy.sh/users/4023183) & [Sonnyc](https://osu.ppy.sh/users/11771)
- **BN 4 Year Badge:**  [Nozhomi](https://osu.ppy.sh/users/2716981)
- **QAT/NAT 1 Year Badge:** [Ascendance](https://osu.ppy.sh/users/2931883), [Noffy](https://osu.ppy.sh/users/1541323) & [Faputa](https://osu.ppy.sh/users/845733)
- **QAT/NAT 2 Year Badge:** [Lasse](https://osu.ppy.sh/users/896613), [-Mo-](https://osu.ppy.sh/users/2202163) & [Chaoslitz](https://osu.ppy.sh/users/3621552)
- **QAT 3 Year Badge:** [OnosakiHito](https://osu.ppy.sh/users/290128)

## <a id="mappers-showcase"></a>Mappers' Showcase

> The Mappers' Showcase parades lesser-known mappers from every game mode. Since we bundled two issues in one post, this month will feature two game modes. 

Reviewer 5: ```suggestion
The Mappers' Showcase presents lesser-known mappers from every game mode. Since we bundled two issues in one post, his month we will feature two game modes. 
```

> Thanks to our selector for osu!catch [Jemzuu](https://osu.ppy.sh/users/7890134) and our selectors for osu!mania [Scotty](https://osu.ppy.sh/users/11085809) & [Unpredictable](https://osu.ppy.sh/users/7560872)!

Reviewer 5: ```suggestion
Thanks to [Jemzuu](https://osu.ppy.sh/users/7890134) for the osu!catch selections and [Scotty](https://osu.ppy.sh/users/11085809) & [Unpredictable](https://osu.ppy.sh/users/7560872) for the osu!mania selections!
```

### Phob (osu!catch)

> Recently entering their way in the gates of the ranked section, **[Phob](https://osu.ppy.sh/users/6069462)** is a newer mapper that is currently having a decent breakout year so far! Although he's more known for being a skillful Hidden player, his mapping definitely deserves more recognition! He crafts his maps with brilliant relevance to the music showing such coherence in patterning, proving that he clearly knows what he's doing!

Reviewer 5: ```suggestion
Having recently entered their way in the gates of the ranked section, **[Phob](https://osu.ppy.sh/users/6069462)** is a new mapper that is currently having a decent breakout year so far! Although he's more known for being a skilled Hidden player, his mapping definitely deserves more recognition! He crafts his maps with brilliant relevance to the music showing such coherence in patterning, proving he clearly knows what he's doing!
```

> His take on [Feint - Snake Eyes (feat, CoMa)](https://osu.ppy.sh/beatmapsets/1034056#fruits/2162020) exemplifies the mapping spirit of clean and consistency-based maps, as throughout his difficulties, they were made of superb quality. For a relatively newer mapper, this set was surely impressive! Moreover, his breakcore map, [Lolipusher - why as soon as we become so close, we have to say goodbye?](https://osu.ppy.sh/beatmapsets/1135515#fruits/2437918) manifests that Phob can still stand out regardless of the song's complexity and genre, while still maintaining the players' interest through his solid patterning!

Reviewer 5: ```suggestion
His take on [Feint - Snake Eyes (feat. CoMa)](https://osu.ppy.sh/beatmapsets/1034056#fruits/2162020) exemplifies the mapping spirit of clean and consistency-based maps, as throughout his difficulties, they were made of superb quality. For a relatively new mapper, this set is surely impressive! Moreover, his breakcore map, [Lolipusher - why as soon as we become so close, we have to say goodbye?](https://osu.ppy.sh/beatmapsets/1135515#fruits/2437918) manifests that Phob can still stand out regardless of song's complexity and genre, while still maintaining players' interest through solid patterning!
```

> He also tends to favor mapping lower difficulties, especially Salads, so he is certainly a mapper that is in good shape! We are definitely going to watch out for his future works, as he is a very promising mapper! If novice mappers are looking for an inspiration, Phob would be a great example!

Reviewer 5: ```suggestion
Phob also tends to favor mapping lower difficulties, especially Salads, so he is certainly a mapper in good shape! We are definitely going to look forward to his future works, as he is a very promising mapper! If novice mappers are looking for an inspiration, Phob is definitely a great example!
```

### dika312 (osu!catch)

> A long-term mapper that has been crafting maps for almost 5 years, **[dika312](https://osu.ppy.sh/users/741613)** is someone that strives for an intuitive yet a very interesting mapping style that would keep you on the edge of your seat! Despite his long mapping experience, none of dika's works has reached the ranked section (yet!), and most of his maps are sadly gone unnoticed.

Reviewer 5: ```suggestion
A long-term mapper that has been crafting maps for almost 5 years, **[dika312](https://osu.ppy.sh/users/741613)** is someone that strives for an intuitive yet a very interesting mapping style that will keep you on the edge of your seat! Despite his long mapping experience, none of dika's works has reached the ranked section (yet!), and most of his maps are sadly unnoticed by the public.
```

> A diamond in the rough are his maps such as [Kikuo - Neko no Shokutaku](https://osu.ppy.sh/beatmapsets/839016#fruits/1756467), where he demonstrated his virtuosity in movement and exquisite patterning to create a gratifying piece of art on such a lovely song. Another solid example would be his rendition of [MALVA. - Trane](https://osu.ppy.sh/beatmapsets/1010141#fruits/2114540), wherein he showed his proficiency in varying platter-motion once again — making use of diverse flow and patterns, leaving a pleasurable experience and taste for mappers and players alike.

Reviewer 5: ```suggestion
A diamond in the rough are his maps such as [Kikuo - Neko no Shokutaku](https://osu.ppy.sh/beatmapsets/839016#fruits/1756467), in which he has demonstrated his virtuosity in movement and exquisite patterning to create a gratifying piece of art from such a lovely song. Another solid example would be his rendition of [MALVA. - Trane](https://osu.ppy.sh/beatmapsets/1010141#fruits/2114540), wherein he has shown his proficiency in varying platter-motion once again — making use of diverse flow and patterns, leaving a pleasurable experience and taste for both mappers and players.
```

> Other of his great works that is worth mentioning would be his take on [KikuoHana - Nobore! Susume! Takai Tou](https://osu.ppy.sh/beatmapsets/545757#fruits/2084234), showcasing exceptional progression in style as the song intensifies, along with fine contrast in calmer sections. We hope to see more of dika's maps, and hopefully he can bear us good fruit in the ranked section someday!

Reviewer 5: ```suggestion
Another of his great works that is worth mentioning would be his take on [KikuoHana - Nobore! Susume! Takai Tou](https://osu.ppy.sh/beatmapsets/545757#fruits/2084234), showcasing exceptional progression in style as the song intensifies, along with fine contrast in calmer sections. We hope to see more of dika's maps, and hopefully he can bear us good fruit in the ranked section someday!
```

An example of a map which culminates this is their rendition of [Sta - 99 Glooms](https://osu.ppy.sh/beatmapsets/961775#mania/2013908). Its liberal usage of flams and more detailed interpretation of the track's various piano and synth are what make this map really shine. Another notable works of theirs is, [sasakure.UK - Jack-the-Ripper](https://osu.ppy.sh/beatmapsets/1130177#mania/2360897), a map featuring complex timing and even more complex representation of the track's guitar sounds. We hope to see Ticle's maps gain more traction as they are definitely worth a shot for any LN enthusiast!

## <a id="mapping-contests"></a>Mapping Contests

> Get your editor ready. The mapping contests are here! Whether you are a veteran or making your first steps in mapping, contests are for everyone! In this section you can see what contests have been held in May and June or are ongoing!

Reviewer 5: ```suggestion
Get your editor ready. The mapping contests are here! Whether you are a veteran or making your first steps in mapping, contests are for everyone! In this section you can see what contests were held in May and June or are ongoing!
```

### Monthly Beatmapping Contests

> The [Monthly Beatmapping Contests](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest) (MBC) gives to you your monthly dose of mapping competitions. Each month's contest features a different mapping limitation that entries must adhere to.

Reviewer 5: ```suggestion
The [Monthly Beatmapping Contests](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest) (MBC) give you your monthly dose of mapping competitions. Each month's contest features a different mapping limitation that entries must adhere to.
```

> The April's contest results are out and we congratulate [hypercyte](https://osu.ppy.sh/users/9155377) for winning and getting his Tier 2 badge! Good job also to [fanzhen0019](https://osu.ppy.sh/users/418699) for being community vote runner-up and [Acylica](https://osu.ppy.sh/users/1943309) for being the screening crew favorite!

Reviewer 5: ```suggestion
The April's contest results are out and we congratulate [hypercyte](https://osu.ppy.sh/users/9155377) for winning and getting his Tier 2 badge! Good job to [fanzhen0019](https://osu.ppy.sh/users/418699) as well for being the community vote runner-up and [Acylica](https://osu.ppy.sh/users/1943309) for being the screening crew favorite!
```

> The May and June Montly Beatmapping Contests have both of their submission phases concluded and we hope to see the results soon. Once they are out we'll be sure to let you know!

Reviewer 5: ```suggestion
The May and June Montly Beatmapping Contests both have their submission phases concluded and we hope to see the results soon. Once they are out, we'll be sure to let you know!
```

### osu! Mapping Olympiad #7: osu!mania

> Keyboard mashers, rise up! It's your time to shine!
> The [osu! Mapping Olympiad](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) is a returning mapping contest which features a different game mode with every version. And this time only for the osu!mania mappers out there!

Reviewer 5: ```suggestion
Keyboard mashers, rise up! It's your time to shine!

The [osu! Mapping Olympiad](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) is a returning mapping contest which features a different game mode with every version. And this time only for the osu!mania mappers out there!
```

> Keyboard mashers, rise up! It's your time to shine!
> The [osu! Mapping Olympiad](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) is a returning mapping contest which features a different game mode with every version. And this time only for the osu!mania mappers out there!

Reviewer 5: ```suggestion
Keyboard mashers, rise up! It's your time to shine!
The [osu! Mapping Olympiad](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) is a returning mapping contest which features a different game mode in every installment. This time only for the osu!mania mappers out there!
```

Luring all mappers out there with Olympiad points for the top 3, which awards the Mapping Olympiad badge, and for the winner an unique user title, the "Elite Mapper"!

> Do you have what it takes to be the next "Elite Mapper"? Then jump right into it, grab one of the pre-timed .osz and start to map! We're excited to see all the mania maps!

Reviewer 5: ```suggestion
Do you have what it takes to become the next "Elite Mapper"? Then jump right into it, grab one of the pre-timed .osz and start mapping! We're excited to see all your maps!
```

> ### Aspirve V

Reviewer: ```suggestion
### Aspire V
```

> The Category Stage Voting of the Asprive V mapping contest is currently open! Out of all submissions, 9 unique categories were created with each having 5 hand picked beatmaps. Make sure to check all of them out and vote for your favorite. You can find the categories, links to the voting pages and downloads in the [Aspire V - Category Stage Voting](https://osu.ppy.sh/home/news/2020-06-22-aspire-v-category-stage-voting) newspost.

Reviewer 5: ```suggestion
The Category Stage Voting of the Aspire V mapping contest is currently open! Out of all submissions, 9 unique categories were created with each having 5 hand picked beatmaps. Make sure to check all of them out and vote for your favorites. You can find the categories, links to the voting pages and downloads in the [Aspire V - Category Stage Voting](https://osu.ppy.sh/home/news/2020-06-22-aspire-v-category-stage-voting) newspost.
```

> But make sure to hurry, only roughly a week is left! 

Reviewer 5: ```suggestion
Make sure to hurry up, there's roughly a week left! 
```

### 2020 osu!catch Mapping World Cup

Fruit catchers, now's your chance! The [2020 osu!catch Mapping World Cup](https://osu.ppy.sh/community/forums/topics/1089125) has arrived! In it's 2nd edition, the best osu!catch mappers from ten countries will compete against each other in a round based elimination format to decide who is the best of the best. Will the Philippines be able to defend their title or will we see a new country as the osu!catch mapping champion? 

We will surely keep you up to date! 

### lu^3 Mapping Cup

> Three time's the charm! The [lululu Mapping Cup](https://osu.ppy.sh/community/forums/topics/1086878) with it's 3rd edition has returned! This is a mapping contest for the osu! game mode only, featuring a specific song theme with every contest! This year's contest is using the "Swing Beat" theme.

Reviewer 5: ```suggestion
Third time's the charm! The [lululu Mapping Cup](https://osu.ppy.sh/community/forums/topics/1086878) has returned  with its 3rd edition! This is a mapping contest for the osu! game mode only, featuring a specific song theme in every installment! This year's contest is using the "Swing Beat" theme.
```

> This contests has also something special. It can have two winners! One winner will be selected from the Taiwanese participants, the other winner will be selected by anyone. That's right, everyone is allowed to participate in the contest! 

Reviewer 5: ```suggestion
This contest also has something special. There can be two winners! One winner will be selected from the Taiwanese participants, the other winner will be selected from every contestant. That's right, everyone is allowed to participate in this contest! 
```

> So what are you waiting for, grab the pre-timed .osz and join in! Supporter and maybe even a profile badge is waiting for you!

Reviewer 5: ```suggestion
So what are you waiting for, grab the pre-timed .osz and join in! osu!supporter tag and maybe even a profile badge is waiting for you!
```

> Are there any other mapping contests going on? Or being announced soon? Please let us know! We would love to give it more attention to the public. No matter if it's mode or language specific. Head over to the [forum post](https://osu.ppy.sh/community/forums/topics/1027590) where you can find the submission form for it. 

Reviewer 5: ```suggestion
Are there any other mapping contests going on? Or being announced soon? Please let us know! We would love to give them more attention to the public. No matter if it's mode or language specific. Head over to the [forum post](https://osu.ppy.sh/community/forums/topics/1027590) where you can find the submission form for them. 
```

Third time's the charm! The [lululu Mapping Cup](https://osu.ppy.sh/community/forums/topics/1086878) has returned  with its 3rd edition! This is a mapping contest for the osu! game mode only, featuring a specific song theme in every installment! This year's contest is using the "Swing Beat" theme.
This contest also has something special. There can be two winners! One winner will be selected from the Taiwanese participants, the other winner will be selected from every contestant. That's right, everyone is allowed to participate in this contest! 

So what are you waiting for, grab the pre-timed .osz and join in! osu!supporter tag and maybe even a profile badge is waiting for you!

Are there any other mapping contests going on? Or being announced soon? Please let us know! We would love to give them more attention to the public. No matter if it's mode or language specific. Head over to the [forum post](https://osu.ppy.sh/community/forums/topics/1027590) where you can find the submission form for them. 

We hope to see a lot of exciting mapping contests in the future!

## <a id="around-community"></a>Around the Community

### New Review System in Beatmap Discussion

> Modders, listen. We have great news for you! As some may already noticed, modding got an update! The review system which got announced a while ago got finally added to the Beatmap Discussion! What it is you ask?

Reviewer 5: ```suggestion
Modders, listen. We have great news for you! As some may have already noticed, modding system got an update! The review system which was announced a while ago got finally added to the Beatmap Discussions! What is it you ask?
```

> Some may remember the old modding system which worked through the forum. People posted mods in mappers' beatmap forum threads. The difference to the current modding system is that modder could bundle everything in one, huge post if they liked. A lot of modders liked it and still would like to have a similar format when modding.

Reviewer 5: ```suggestion
Some may remember the old modding system which worked through the forum. People posted mods in mappers' beatmap forum threads. The difference to the current modding system is that modder could bundle everything in one, huge post if they liked. A lot of modders liked it and would still prefer to have a similar format while modding.
```

Now they finally do! You can now bundle your mods in one, big post! To access it, go into the beatmap discussion of any beatmap you like to mod and then go to the "Review" tab. That's all! Now you can mod in a format that lets you control what order the mod is read in, which is very useful for reoccurring or bigger issues in maps. Be sure to check out all the features which the current modding system offers to you!

> Do you think it could be better and something is still missing? Or did you find a bug in it? The developers would love to hear your feedback! Come join the [osu!dev Discord server](https://discord.gg/ppy) and join the discussion about the review system in the #modding channel! 

Reviewer 5: ```suggestion
Do you think it could be better or something is still missing? Or did you find a bug in it? The developers would love to hear your feedback! Come join the [osu!dev Discord server](https://discord.gg/ppy) and join the discussion about the review system in the #modding channel! 
```

### The osu!wiki is hiring

> Okay, hiring is maybe the wrong word for it. But on a serious note, the osu!wiki is a huge knowledge database which features topics about everything about osu! Not just mapping and/or modding related such as the entire Ranking Criteria or various guides for mapping, but also all kinds of information, such as the osu! team, various gameplay guides, contests and tournaments and much more.

Reviewer 5: ```suggestion
Okay, hiring is maybe the wrong word for it. But on a serious note, the osu!wiki is a huge knowledge base which features articles all about osu!, Not only mapping or modding related content such as the entire Ranking Criteria, but also all kinds of information, such as the osu! team, various gameplay guides, contests and tournaments and much more.
```

> But sadly it's run by only a few people and a lot of wiki entries are outdated. But good news: Everyone can help! And the people behind the osu!wiki would be surely happy to see more helping hands to update everything and provide the latest information all around osu! 

Reviewer 5: ```suggestion
But sadly it's run by only a few people and a lot of wiki entries are outdated. But good news: anyone can help! And the people behind the osu!wiki will be surely happy to see more helping hands to update everything and provide the latest information all around osu!.
```

Contributing is not even that hard and you can get all help you need to get started in the #osu-wiki channel on the [osu!dev Discord server](https://discord.gg/ppy). Do not hesitate to ask there, they won't bite you!

<a id="conclusion"></a>

---

> Phew, that was a bit of a read, right? We're happy that you made it until here! Once again sorry for the long wait. We will work on our schedule to make sure future issues of the Mappers' Report going out in time. You can expect us in the future with the end of every month. Should really not that much happen in osu! we will probably make again one bigger news post, but we try to keep these on a minimum! And please don't forget, should you have anything interesting and mapping or modding related for us to mention, make sure to tell [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323)!

Reviewer 5: ```suggestion
Phew, that was a bit of a read, right? We're happy that you made it until here! Once again sorry for the long wait. We will work on our schedule to make sure future issues of the Mappers' Report get released in time. You can expect us in the future at the end of every month. Should really not that much happen in osu!, we will probably make again one bigger news post, but we try to keep these on a minimum! And please don't forget, should you have anything interesting and mapping or modding related for us to mention, make sure to contact [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323)!
```
